### PR TITLE
Remove schemas and datasets

### PR DESCRIPTION
**Pull request by otfrom**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -14,11 +14,9 @@
                  [kixi/stats "0.4.3"]
                  [me.raynes/fs "1.4.6"]
                  [medley "1.0.0"]
-                 [net.mikera/core.matrix "0.55.0" :exclusions [org.clojure/clojure]] ;; going to 0.62.0 is bad
                  [org.apache.commons/commons-math3 "3.6.1"]
                  [org.hdrhistogram/HdrHistogram "2.1.9"] ;; upgrading to 2.1.10 causes a test to fail
-                 [prismatic/schema "1.1.9"]
-                 [schema-contrib "0.1.5"]]
+                 ]
```
Comment (seb231) [inline project.clj]: remove extra line

File: src/clj/witan/send/main.clj [hunk 2]
```diff
@@ -5,16 +5,16 @@
...
-                   :transitions sc/TransitionCounts
-                   :population sc/PopulationDataset
-                   :costs sc/NeedSettingCost
-                   :valid-states sc/ValidSettingAcademicYears}})
+  {:schema-inputs #_{:settings-to-change sc/SettingsToChange
+                     :transitions sc/TransitionCounts
+                     :population sc/PopulationDataset
+                     :costs sc/NeedSettingCost
+                     :valid-states sc/ValidSettingAcademicYears}
+   {}})
```
Comment (seb231) [inline src/clj/witan/send/main.clj]: why the additional empty map?

Comment (seb231) [reply src/clj/witan/send/main.clj]: ah don't worry this is the "other bits of old validation hanging around" could you add comments for these please

File: src/clj/witan/send/model/input.clj [hunk 3]
```diff
@@ -1,69 +1,35 @@
...
+        x
+        (int? x)
+        (double x)
+        (string? x)
+        (Double/valueOf x)
+        :else
+        (throw (ex-info (format "Failed to parse supplied value '%s'" x)
+                        {:value x}))))
+
+(defn csv->x [xf file-name]
```
Comment (seb231) [inline src/clj/witan/send/model/input.clj]: does `x` indicate something I'm unaware of?

File: src/clj/witan/send/send.clj [hunk 4]
```diff
@@ -1,24 +1,42 @@
...
-     (if (= true validate-input)
-       (-> (p/prepare-send-inputs input (:transition-parameters config))
-           (r/run-send-model (:projection-parameters config)))
-       validate-input))))
+   (let [input (build-input-datasets (:project-dir config) (:file-inputs config) (:schema-inputs config))
+         ;; validate-input (i/check-if-dataset-is-valid input)
+         ]
+     (-> (p/prepare-send-inputs input (:transition-parameters config))
+         (r/run-send-model (:projection-parameters config)))
+     #_(if (= true validate-input)
```
Comment (seb231) [inline src/clj/witan/send/send.clj]: quick comment here would be appreciated

File: test/witan/send/acceptance/model_test.clj [hunk 5]
```diff
@@ -51,24 +51,61 @@
...
+        (is (= (find-in-map checking-leaver-exp :ay "11")
+               (find-in-map scenario-leaver-exp :ay "11")))
+        (is (= (find-in-map checking-mover-exp :ay "11")
+               (find-in-map scenario-mover-exp :ay "11")))))))
+
+
+(comment
+
+  ;; find out why the above test is failing
+  ;; I think I've got waaaay to many invalid states which makes me think I'm not passing through the right data somewhere.
```
Comment (seb231) [inline test/witan/send/acceptance/model_test.clj]: nope that's to be expected here

File: src/clj/witan/send/model/input.clj (hunk 3 above)
Comment (otfrom) [reply src/clj/witan/send/model/input.clj]: x means anything here as it is just doing the dumb work of the zipmap. All of the rest of the scrubbing and formatting is done in the sub namespaces

File: src/clj/witan/send/send.clj (hunk 4 above)
Comment (otfrom) [reply src/clj/witan/send/send.clj]: done

File: test/witan/send/acceptance/model_test.clj (hunk 5 above)
Comment (otfrom) [reply test/witan/send/acceptance/model_test.clj]: removed

File: project.clj (hunk 1 above)
Comment (otfrom) [reply project.clj]: done

File: src/clj/witan/send/main.clj (hunk 2 above)
Comment (otfrom) [reply src/clj/witan/send/main.clj]: commented

**Pull request merged**